core-bundle/src/Resources/contao/classes/Backend.php

```diff
@@ -537,19 +539,17 @@ protected function getBackendModule($module, PickerInterface $picker = null)
 							$trail[] = ' <span>' . $GLOBALS['TL_LANG']['MOD'][$table] . '</span>';
 						}
 
+						$strUrl = $container->get('router')->generate('contao_backend', array
+						(
+							'do' => $container->get('request_stack')->getCurrentRequest()->query->get('do'),
+							'table' => $table,
+							'id' => $objRow->id,
+							'ref' => $container->get('request_stack')->getCurrentRequest()->attributes->get('_contao_referer_id'),
+							'rt' => REQUEST_TOKEN,
+						));
+
 						// Add object title or name
-						if ($objRow->title)
-						{
-							$trail[] = ' <span>' . $objRow->title . '</span>';
-						}
-						elseif ($objRow->name)
-						{
-							$trail[] = ' <span>' . $objRow->name . '</span>';
-						}
-						elseif ($objRow->headline)
-						{
-							$trail[] = ' <span>' . $objRow->headline . '</span>';
-						}
+						$trail[] = sprintf(' <span><a href="%s">%s</a></span>', $strUrl, ($objRow->title ?: $objRow->name ?: $objRow->headline));
```

Comment:
This previously did not add a breadcrumb if the record neither had a title, name or headline. I don't think the current behaviour is correct, maybe we should fall back to the ID or not change the behaviour?

Reply:
I think not showing anything is an issue. Actually the field should be configurable and just fall back to one of those three but I wouldn't know where to put that config in the DCA. And then fall back to the ID maybe, yes?

Reply:
But it could still be merged as is because ignoring it like we did for years cannot be the desired behaviour? :D

Reply:
I am also not sure about this. Even though it might be a bug, I would prefer if we did not change it now. 🙈

Reply:
Adjusted in https://github.com/contao/contao/pull/3727/commits/1799660d1fbdc79c7b9bf88ed092916bf33e653c then.

Reply:
Excellent solution. 👍